scripts/makei

```diff
@@ -139,8 +141,37 @@ def add_init_parser(subparsers):
                              action='store_true')
     init_parser.set_defaults(handle=handle_init)
 
+def add_cvtsrcpf_parser(subparsers: argparse.ArgumentParser):
+    cvtsrcpf_parser = subparsers.add_parser(
+        'cvtsrcpf',
+        help='convert source physical file members to ASCII IFS files',
+        description='Converts all members in a source physical file to properly-named \
+                        (Better Object Builder-compatible), ASCII-ish, LF-terminated source files \
+                        in the current directory in the IFS. An .ibmi.json will also be created at the same directory.'
+    )
+
+    cvtsrcpf_parser.add_argument(
+        "file",
+        help='the name of the source file',
+        metavar='<file>',
+    )
+
+    cvtsrcpf_parser.add_argument(
+        "library",
+        help='the name of the library',
+        metavar='<library>',
+    )
+
+    cvtsrcpf_parser.add_argument(
+        "-c",
+        help='Output the joblog to the specified json file.',
```

Comment:
this description is not about CCSID as it should be
the EBCDIC CCSID that the source in this directory should be compiled with